src/content/plugins/chunks-webpack-plugin.md

```diff
@@ -0,0 +1,322 @@
+
+## Zero config
+
+The plugin works without configuration. For advanced usage, see the section [using configuration](#using-a-configuration).
+
+## Installation
+
+The plugin is available as a package with the name of `chunks-webpack-plugin` on [npm](https://www.npmjs.com/package/chunks-webpack-plugin) and [Github](https://github.com/yoriiis/chunks-webpack-plugin).
+
+```bash
+npm install chunks-webpack-plugin --save-dev
+```
+
+```bash
+yarn add chunks-webpack-plugin --dev
+```
+
+## Environment
+
+ChunksWebpackPlugin was built for Node.js `>=8.11.2` and webpack `>=4.x`.
+
+## Example
+
+For example implementation, please see the [Github documentation](https://github.com/yoriiis/chunks-webpack-plugin/blob/master/README.md#example).
+
+## Basic usage
+
+The plugin will generate two HTML files for each entry points. Each filename contains the entry point name, the `{{entry}}` placeholder is automatically replaced.
+
+- `{{entry}}-styles.html`: contains all HTML `<link>` tags
+- `{{entry}}-scripts.html`: contains all HTML `<script>` tags
+
+First, let's add the plugin to the webpack configuration.
+
+```js
+var ChunksWebpackPlugin = require('chunks-webpack-plugin');
+var path = require('path');
+
+module.exports = {
+  entry: 'main.js',
+  output: {
+    filename: 'main.js',
+    path: path.resolve(__dirname, './dist')
+  },
+  plugins: [new ChunksWebpackPlugin()]
+};
+```
+
+HTML files are built in the output path directory with all the other assets.
+
+Then, include the HTML files in the wanted pages.
+
+__main-styles.html__
+
+```html
+<link rel="stylesheet" href="main.css" />
+```
+
+__main-scripts.html__
+
+```html
+<script src="main.js"></script>
+```
+
+---
+
+## Using a configuration
+
+You can pass configuration options to ChunksWebpackPlugin to overrides default settings. Allowed values are as follows:
+
+### `outputPath`
+
+`string = null`
+
+Tells the plugin whether to personalize the output path of the generated files.
+
+```js
+new ChunksWebpackPlugin({
+  outputPath: path.resolve(__dirname, './templates')
+});
+```
+
+T> The default behavior will use the  `options.output.path` from the [webpack configuration](https://webpack.js.org/configuration/output/#outputpath).
+
+W> The `outputPath` must be an absolute path.
+
+### `fileExtension`
+
+`string = '.html'`
+
+Tells the plugin whether to personalize the extension of the generated files.
+
+```js
+new ChunksWebpackPlugin({
+  fileExtension: '.html'
+});
+```
+
+### `templateStyle`
+
+`string = '<link rel="stylesheet" href="{{chunk}}" />'`
+
+Tells the plugin whether to personalize the default template for the HTML `<style>` tags. For example, add additional attributes, a CDN prefix or something else.
+
+```js
+new ChunksWebpackPlugin({
+  templateStyle: '<link rel="stylesheet" href="{{chunk}}" />'
```

Comment:
Please provide a meaningful example, not the default value 
```suggestion
  templateStyle: '<link rel="stylesheet" href="{{chunk}}" />'
```

Reply:
I have add a CDN prefix in this [commit](https://github.com/webpack/webpack.js.org/pull/3771/commits/8cac60f7c3a1a6cfb4def7be6962790b4135c294)